app/models/chart_data.rb

```diff
@@ -15,6 +17,15 @@ def data
     [values]
   end
 
+  def success?
+    ! data.first.series_data.nil?
+  rescue EnergySparksNotEnoughDataException
```

Comment:
Possible we should catch these too? https://github.com/BathHacked/energy-sparks/blob/master/app/services/equivalences/calculator.rb#L27

Reply:
Yes, good point